newbank/server/NewBankClientHandler.java

```diff
@@ -7,42 +7,149 @@
 import java.net.Socket;
 
 public class NewBankClientHandler extends Thread{
-	
+
 	private NewBank bank;
 	private BufferedReader in;
 	private PrintWriter out;
-	
-	
+
 	public NewBankClientHandler(Socket s) throws IOException {
 		bank = NewBank.getBank();
 		in = new BufferedReader(new InputStreamReader(s.getInputStream()));
 		out = new PrintWriter(s.getOutputStream(), true);
 	}
-	
+
+	//create user method
+	public void createUser() {
+		try {
+			out.println("Please create a new username:");
+			// reads the user input
+			String userName = in.readLine();
+			// new object for new customer
+			Customer newCustomer = new Customer();
+			// create a boilerplate account for the customer
+			newCustomer.addAccount(new Account("Main", 00.0));
+			// adds  customer to the list of existing customers and assigns them a username
+			bank.customers.put(userName, newCustomer);
```

Comment:
The field is private so I got an error here. 
So maybe we should use a setter method .addCustomer() which does "customers.put(userNamer, newCustomer)" ?